indico/htdocs/js/indico/modules/eventdisplay.js

```diff
@@ -23,6 +23,51 @@ $(document).ready(function() {
         });
     });
 
+    var gradientLayer = $('.participant-list-wrapper > .gradient-layer');
+    var heightControl = $('<div>').addClass('height-control');
```

Comment:
like this you create it with the class instead of adding it afterwards. your way is not completely wrong either. mostly a matter of taste